### PR TITLE
Add a reference link

### PR DESCRIPTION
**Pull request by imba-tjd**

### CODE REVIEW

File: docs/standard/generics/index.md [hunk 1]
```diff
@@ -109,7 +109,7 @@ ms.author: "mairaw"
...
   
--   Generics streamline dynamically generated code. When you use generics with dynamically generated code you do not need to generate the type. This increases the number of scenarios in which you can use lightweight dynamic methods instead of generating entire assemblies. For more information, see How to: Define and Execute Dynamic Methods and DynamicMethod.  
+-   Generics streamline dynamically generated code. When you use generics with dynamically generated code you do not need to generate the type. This increases the number of scenarios in which you can use lightweight dynamic methods instead of generating entire assemblies. For more information, see [How to: Define and Execute Dynamic Methods and DynamicMethod](../../../docs/framework/reflection-and-codedom/how-to-define-and-execute-dynamic-methods.md).
```
Comment (pkulikov) [inline docs/standard/generics/index.md]: @imba-tjd good catch! 
I was puzzled why the link text doesn't match the article title. Most probably, there are two links here:
- one you've added to **How to: Define and Execute Dynamic Methods**
- another one should be the xref link (`<xref:System.Reflection.Emit.DynamicMethod>`) to https://docs.microsoft.com/en-us/dotnet/api/system.reflection.emit.dynamicmethod

What do you think?

Comment (imba-tjd) [reply docs/standard/generics/index.md]: Yeah... I think you are right. Thanks, I will try to modify as you said.

Comment (pkulikov) [reply docs/standard/generics/index.md]: Cool! Thanks!

**Review by rpetrusha — APPROVED** [review]
Thanks for adding these links, @imba-tjd. I'll merge your PR now. Your changes should be live on docs.microsoft.com in the next day or two.

**Pull request merged**